app/operations/assignments/create.rb

```diff
@@ -1,7 +1,7 @@
 module Assignments
   class Create < Operation
-    integer :base_project_id, default: Rails.application.secrets["zooniverse_project_id"]
```

Comment:
Does this mean that API clients (i.e. the current Wildcam Labs frontend) need to pass these in? Or is that already being done? /cc @shaunanoordin